README.md

```diff
@@ -137,6 +137,8 @@ If you are willing to contribute, open a pull request to complete, update or fil
 
 ### Q
 
+* [QUnit](/glossary/QUNIT.md): *to be completed.*
```

Comment:
You may use something like: "A JavaScript unit testing framework maintained by the jQuery Foundation and used on jQuery, Ember, LoDash, Backbone, etc.